src/IssueViz.Security/Models/IHotspot.cs

```diff
@@ -82,9 +73,9 @@ internal class Hotspot : IHotspot
         public int StartLineOffset { get; }
         public int EndLineOffset { get; }
         public string LineHash { get; }
-        public string RuleKey { get; }
+        public string RuleKey => Rule.RuleKey;
```

Comment:
`IAnalysisIssueBase` has a `RuleKey` property so `IHotspot` has to provide an implementation.
However, the new `IHotspotRule` interface has a `RuleKey` property, so the `IHotspot` implementation can just delegate to it.